lib/accountregistry.h

```diff
@@ -52,7 +52,8 @@ class QUOTIENT_API AccountRegistry : public QAbstractListModel,
     const_reference front() const { return vector_t::front(); }
     const_reference back() const { return vector_t::back(); }
     bool isLoggedIn(const QString& userId) const;
-    Connection* get(const QString& userId);
+    Connection* get(const QString& userId); // FIXME: remove in 0.8
```

Comment:
[[deprecated]]?

Reply:
I actually tried to deprecate it, only to find misleading warnings whenever I use `Accounts.get()` or an equivalent :-\ I'm actually unsure if adding `const` to the signature would even break the API/ABI compat - if not, I'd rather just apply the change and retain only one signature.

Reply:
It would be an ABI break

Reply:
Ok, then I'm not hallucinating and it better stay like this for now.